resources/apiserver-proxy/files/xip-certificate.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
```

Comment:
This introduces dependency to the certificates module. Cert-manager CRDs are not installed in cluster essentials. The apiserver-module installation can fail if cert-manager CRDs are not installed yet.

Reply:
I am aware of this dependency, but we cannot install more CRDs in cluster-essentials. The chart is reaching its maximum data capacity